less/mixins.less

```diff
@@ -1 +1,26 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+.nav-link(){
+    font: @nav-font;
+    text-decoration: none;
+    color: @main-font-color;
+}
+
+.otherimg(){
+    border-radius: .5rem;
```

Comment:
Mixins should encorporate 2 or more styles to achieve reusability/DRY.  If it's just 1 style, I'd consider making it a variable.  That way it's not as obsure what `.otherimg()` means